bundles/org.openhab.binding.carnet/src/main/java/org/openhab/binding/carnet/internal/handler/CarNetVehicleHandler.java

```diff
@@ -0,0 +1,643 @@
+                for (Map.Entry<String, CarNetVehicleBaseService> s : services.entrySet()) {
+                    s.getValue().createChannels(channels);
+                }
+
+                logger.debug("{}: Creating {} channels", thingId, channels.size());
+                ArrayList<ChannelIdMapEntry> channelList = new ArrayList<>(channels.values());
+                /*
+                 * try (FileWriter myWriter = new FileWriter("carnetChannels.MD")) {
+                 * String lastGroup = "";
+                 * for (ChannelIdMapEntry e : channelList) {
+                 * String group = lastGroup.equals(e.groupName) ? "" : e.groupName;
+                 * String s = String.format("| %-12.12s | %-23.23s | %-20.20s | %-7s | %-85s |\n", group,
+                 * e.channelName, e.itemType, e.readOnly ? "yes" : "no", e.getDescription());
+                 * myWriter.write(s);
+                 * lastGroup = e.groupName;
+                 * }
+                 * } catch (IOException e) {
+                 * }
+                 */
+                createChannels(channelList);
+                channelsCreated = true;
+            }
+        } catch (CarNetException e) {
+            CarNetApiErrorDTO res = e.getApiResult().getApiError();
+            if (res.description.contains("disabled ")) {
+                // Status service in the vehicle is disabled
+                String message = "Status service is disabled, check data privacy settings in MMI: " + res;
+                logger.debug("{}: {}", thingId, message);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, message);
+                return false;
+            }
+
+            successful = false;
+            error = getError(e);
+        } catch (RuntimeException e) {
+            error = "General Error: " + getString(e.getMessage());
+            logger.warn("{}: {}", thingId, error, e);
+        }
+
+        if (!error.isEmpty()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, error);
+        } else {
+            // updateStatus(ThingStatus.ONLINE);
+        }
+        return successful;
+    }
+
+    /**
+     * Brigde status changed
+     */
+    @Override
+    public void stateChanged(ThingStatus status, ThingStatusDetail detail, String message) {
+        forceUpdate = true;
+    }
+
+    @Override
+    public void informationUpdate(@Nullable List<CarNetVehicleInformation> vehicleList) {
+        forceUpdate = true;
+        channelsCreated = false;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            return;
+        }
+
+        ThingStatus s = getThing().getStatus();
+        if ((s == ThingStatus.INITIALIZING) || (s == ThingStatus.UNKNOWN)) {
+            logger.info("{}: Thing not yet fully initialized, command ignored", thingId);
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug.

Please check all.